backend/corpora/dataset_submissions/app.py

```diff
@@ -44,30 +44,30 @@ def dataset_submissions_handler(s3_event: dict, unused_context) -> None:
         logger.debug(parsed)
```

Comment:
@Bento007  Above on ln 41, and below on lns 89 and 112: is there any reason to not change the comments/logging from "uuid" to "id" as well?